addon/v1alpha1/types_clustermanagementaddon.go

```diff
@@ -93,6 +96,8 @@ type ConfigCoordinates struct {
 // ConfigGroupResource represents the GroupResource of the add-on configuration
 type ConfigGroupResource struct {
 	// group of the add-on configuration.
+	// +required
+	// +kubebuilder:default=""
```

Comment:
is this needed? if you set required, default is not necessary, isn't it?

Reply:
mark the group to `option`, but we need to keep `default`, because if one field is marked to `listMapKey` , it must have a default or be a required